docs/sources/operators-guide/configuring/configuring-out-of-order-samples-ingestion.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Configuring out-of-order samples ingestion"
+menuTitle: "Configuring out-of-order samples ingestion"
+description: "Learn how to configure Grafana Mimir to handle out-of-order samples ingestion."
+weight: 120
+---
+
+# Configuring out-of-order samples ingestion
+
+Grafana Mimir and the Prometheus TSDB understand out-of-order as follows.
+
+The moment that a new series sample arrives, Mimir need to determine if the series already exists, and whether or not the sample is too old:
+
+- If the series exists, incoming sample must have newer timestamp than the latest sample that is stored for the series.
+  Otherwise, it is considered out-of-order and will be dropped by the ingesters.
+- If the series does not exist, then the sample has to be within bounds, which go back 1 hour from TSDB's head-block max time (when using 2 hour block range). If it fails to be within bounds, then it is also considered out-of-order and will be dropped by the ingesters.
```

Comment:
> If it fails to be within bounds, then it is also considered out-of-order and will be dropped by the ingesters.

Isn't it considered "out-of-bounds" in that case?

Reply:
It does, my bad. Addressed :+1: